realm/realm-library/src/main/java/io/realm/DynamicRealmObject.java

```diff
@@ -925,6 +926,42 @@ public String toString() {
         return sb.toString();
     }
 
+    /**
+     * Returns {@link RealmResults} containing all {@code srcClassName} class objects that have a relationship
+     * to this object from {@code srcFieldName} field.
+     *
```

Comment:
Also add that an entry is added for each reference, e.g. if the same reference is in a list multiple times, the `src` object will show up here multiple times.